src/timeline/TimelineViewManager.h

```diff
@@ -98,6 +98,8 @@ class TimelineViewManager : public QObject
     void openInviteUsersDialog(InviteesModel *invitees);
     void openProfile(UserProfile *profile);
     void showImagePackSettings(ImagePackListModel *packlist);
+    void openJoinRoomDialog();
+    void joinRoom(QString roomId);
```

Comment:
I think this makes more sense to have in the Nheko object, I really want to get rid of the timeline manager :D

Reply:
OK, will do